madmom/utils/__init__.py

```diff
@@ -419,6 +426,87 @@ def quantize_events(events, fps, length=None, shift=None):
     return quantized
 
 
+def quantize_notes(notes, fps, length=None, num_pitches=None, shift=None,
+                   velocity=None):
+    """
+    Quantize the notes with the given resolution.
+
+    First a sparse 2D array with the length and number of pitches is created.
+    If no `length` and `num_pitches` are given the values are inferred from the
+    notes. Then each note is put into the column corresponding to its pitch
+    and all frames between the note's onset and offset (i.e. onset + duration)
+    are set to the velocity of the note.
```

Comment:
Already much better! I would try to describe **what** the function does, and not **how** it does it, maybe like this:

"Create a sparse 2D array whose rows correspond to points in time (according to `fps` and `length`), and columns to note pitches (according to `num_pitches`). The values of the array correspond to the velocity of a pitch at a given point in time (based on the note pitch, onset, duration, and velocity). If no values for `length` and `num_pitches` are given, they are inferred from `notes`."